bin/chelseawin.rb

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+require ".\\lib\\chelsea.rb"
```

Comment:
I think you want to use `require_relative` here, as once this is deployed out into the ether (RubyGems), it will struggle to find that file, at least from my experience publishing `0.0.1`, lol

Also, I wonder if in `bin/chelsea` you can just do a quick check on Host OS, and switch the require based on that? I don't know for sure if you even need this tho, because I did some quick googling and from what I understand Ruby treats paths identically across platforms?

Reply:
Yeah you're right I think require_relative should be used here and I do believe Ruby treats paths with / correctly according to the platform so a require_relative should work on Linux + Windows and elsewhere.